http_natives.cpp

```diff
@@ -445,6 +509,10 @@ const sp_nativeinfo_t http_natives[] =
 	{"HTTPClient.FollowLocation.set",	SetClientFollowLocation},
 	{"HTTPClient.Timeout.get",			GetClientTimeout},
 	{"HTTPClient.Timeout.set",			SetClientTimeout},
+	{"HTTPClient.MaxSendSpeed.get",		GetClientMaxSendSpeed},
+	{"HTTPClient.MaxSendSpeed.set",		SetClientMaxSendSpeed},
+	{"HTTPClient.MaxRecvSpeed.get",		GetClientMaxRecvSpeed},
+	{"HTTPClient.MaxRecvSpeed.set",		SetClientMaxRecvSpeed},
```

Comment:
You need to add these properties to the `HTTPClient` methodmap in `http.inc` as well.
Preferably with documentation indicating that the value is in bytes/sec.